dataline-integrations/singer/postgres/destination/docker/build.gradle

```diff
@@ -0,0 +1,10 @@
+plugins {
```

Comment:
I don't think it is necessary to go all the way to the docker push here. This is more process piece here, as long as the docker file woks and the tests work (and soon have the config be part of it). 

There is already a push script for the integrations `tools/integrations/manage.sh` which i would love to see become the cli to create the integrations (with scaffolding etc...)

Reply:
Talked offline. We're going to keep this here to build for integration tests and we can use `manage.sh` for actually building and pushing images to an external registry.